i18n/da/building.md

```diff
@@ -0,0 +1,283 @@
+$ sudo pkg_add doxygen graphviz
+```
+
+# Byg seneste boost
+```bash
+# Hent seneste boost
+$ wget [seneste boost] -O latest_boost.tar.bz2
+$ tar xvjf latest_boost.tar.bz2 && cd seneste_boost/
+
+# Download og anvend opdateringer
+# https://svn.boost.org/trac/boost/attachment/ticket/12575/boost-1.62-asio-libressl.patch
+# https://gist.githubusercontent.com/laanwj/bf359281dc319b8ff2e1/raw/92250de8404b97bb99d72ab898f4a8cb35ae1ea3/patch-boost_test_impl_execution_monitor_ipp.patch
+
+# Byg boost
+$ echo 'using gcc : : eg++ : "-fvisibility=hidden -fPIC" "" "ar" "strip"  "ranlib" "" : ;' > user-config.jam
+$ config_opts="runtime-link=shared threadapi=pthread threading=multi link=static variant=release --layout=tagged --build-type=complete --user-config=user-config.jam -sNO_BZIP2=1"
+$ ./bootstrap.sh --without-icu --with-libraries=chrono,log,program_options,date_time,thread,system,filesystem,regex,test
+$ sudo ./b2 -d2 -d1 ${config_opts} --prefix=${BOOST_PREFIX} stage
+$ sudo ./b2 -d0 ${config_opts} --prefix=${BOOST_PREFIX} install
+```
+**Note: se OpenBSD bygge instruktioner nedenunder**
+
+### Windows (MSYS2/MinGW-64)
+* Download [MSYS2 installer](http://msys2.github.io/), 64-bit eller 32 efter dit behov, og kør den.
+* Brug den associerede genvej med din arkitektur for at starte MSYS2 miljøet. På et 64-bit system ville det være MinGW-w64 Win64 Shell shortcut. Bemærk at hvis du kører 64-bit Windows, ville du have både 64-bit og 32-bit miljøer
+* Opdater pakkerne i din MSYS2 installation:
+```
+pacman -Sy
+pacman -Su --ignoregroup base
+pacman -Su
+```
+* For dem af jer der allerede har kendskab til pacman, kan i køre den normale ```pacman -Syu``` for at opdatere, men du kan få fejl og være nødt til at genstarte MSYS2 hvis pacman's afhængigheder er opdateret.
+* Installer afhængigheder: ```pacman -S make mingw-w64-x86_64-cmake mingw-w64-x86_64-gcc mingw-w64-x86_64-boost mingw-w64-x86_64-openssl```
+* Valgfri: ```mingw-w64-x86_64-doxygen```  (Du får brug for [Graphviz](http://graphviz.org/doc/winbuild.html) til doxygen)
+* Note: Du får brug for  ``` mingw-w64-x86_64-miniupnpc``` hvis du er bag en begrænsende NAT firewall.
+
+## Trin 3. Byg
+
+### 1. Klon depotet
+```bash
+$ git clone --recursive https://github.com/monero-project/kovri
+```
+
+### 2. Byg Kovri og undermodul afhængigheder med en kommando
+```bash
+$ make  # for at formindske bygge-tiden, kør `make -j <tilgængelige CPU kerner>`. Eksempel: `make -j4`
+```
+
+### 3. Installer
+```bash
+$ make install
+```
+
+- Slut-brugerne SKAL køre `make install` til nye installationer
+- Udviklere BURDE køre `make install` efter et frisk build
+
+### Andre muligheder du kan bruge i stedet for trin 2:
+
+For en akkurat og komplet liste af muligheder, se targets i [Makefile](https://github.com/monero-project/kovri/blob/master/Makefile).
+
+#### Noter
+- Alle bygge resultater (inkluderet Doxygen) ville være i bygge resultatets mappe
+
+### Clang
+For at bygge med clang, **skal* du eksportere følgende:

```

Comment:
>`**skal*`

You missed ending the word with another `*`. The remaining text is now bold unless you close the word with another `*`.

Reply:
Will fix in a sec :)